scripts/ci/ci_build_dockerhub.sh

```diff
@@ -20,6 +20,7 @@
 # on CI to potentially rebuild (and refresh layers that
 # are not cached) Docker images that are used to run CI jobs
 export FORCE_ANSWER_TO_QUESTIONS="yes"
+export VERBOSE_COMMANDS="true"
```

Comment:
do we really want it to be that verbose?

Reply:
Besides that LGTM

Reply:
Yes. This is only in DockerHub build.  In this case being verbose means that I will see which args were passed to docker build.  It's not a lot in fact - this script only builds the images and not more than that.

I want to keep it enabled for a while before it gets stable. I only found out by accident that tonight build was really not optimized and it would be much faster If I saw the docker build command executed.

Reply:
@feluelle :)